### ISSUE TITLE
[enhancement] Update to the script for character/crew

### ISSUE THREAD

**Issue by jesberpen** [brunocalado/scum-and-villainy-pt-br#2]
Hi Bruno,
Not sure how easy is this to implement. I've tried to add the translation to class in the crew upgrades and items that are not class=playbook or crew type. This of course makes them not to appear in the available list afterwards. Would it be possible to add to the script so it also change the lines for those items and not just the playbook ones?
For now I've reverted and removed the calss on those. Is just a minor thing really, but could be nice to have that field translated.

**Comment by brunocalado** [comment]
I made a macro fix this for portuguese. Do you want it to work if your language?

**Comment by brunocalado** [comment]
This is a old pain. I fixed with my macro. It can be changed to your language.
https://github.com/drewg13/foundryvtt-scum-and-villainy/issues/16

**Comment by jesberpen** [comment]
Hi, I translated your macro to Spanish and it works like a charm (thanks for creating it by the way!). But I did notice if I add the class to items, not the ones relative to playbooks or crew type (like muscle, firedrake, etc.), but the ones like Auxiliary, Weapons, etc. they stop showing too, so I think by translating the class they lose also that connection there. But as I mentioned in the comment, this is just me being picky and wanting all fields translated :D. 
I read that thread, and talked a bit with drewg13 over discord and he mentioned that is not something that is easy to fix, so maybe in a future Foundry release it is possible, otherwise I think is easy to keep it as it is. I asked to add a legend for the crew upgrades that I think is more useful to have in plain sight than opening it in any case :)

**Comment by brunocalado** [comment]
Can you give an example?

What macro version are you? Current is v1.2

**Comment by jesberpen** [comment]
I’m using that version, 1.2. The issue I believe is the same. 

Example: 
take from scum-and-villainy.crew_upgrade.json lines 9-11, you have the AI module, with its name and description. If I add in the field “class”: “Auxiliares” to translate the “auxiliary” from English, so it gets translated from English to Spanish in the system, then the item stops appearing in the list of available upgrades.

As far as I understand the script does a mapping for the classes that correspond to the playbooks and crew types in the game, so not sure if is as simple as adding a mapping for the rest of the classes so the script fixes these too, or is more complex to do. Again, is just a very minor thing, is fine as it is, but is something I found when I saw those fields were missing, and then when I pushed the changes I understood why, then proceeded to remove them. I think it could be fixed the same way in the script, but I’m not sure if it’s worth the effort :D
